pkg/columns/ellipsis/ellipsis_test.go

```diff
@@ -176,9 +178,8 @@ func TestEllipsis(t *testing.T) {
 
 	for _, test := range ellipsisTests {
 		t.Run(fmt.Sprintf("%s_%d@%s", test.Input, test.MaxLength, test.Type.String()), func(t *testing.T) {
-			if res := Shorten([]rune(test.Input), test.MaxLength, test.Type); string(res) != test.Result {
-				t.Errorf("%q (%d): got %q, expected %q", test.Input, test.MaxLength, res, test.Result)
-			}
+			res := Shorten([]rune(test.Input), test.MaxLength, test.Type)
+			require.Equal(t, test.Result, string(res), "%q (%d): got %q, expected %q", test.Input, test.MaxLength, res, test.Result)
```

Comment:
require.Equal should already list what was expected and what we got instead (with a diff even I think)